### PR TITLE
Update oncue.markdown

### PR DESCRIPTION
**Pull request by rssalerno**
Added 20 RESA to list of tested devices.  This model is discontinued with no landing page on the Kohler site, so no link is provided.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @rssalerno, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#23944).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/oncue.markdown [hunk 1]
```diff
@@ -21,6 +21,7 @@ The Oncue by Kohler integration will allow you to monitor the state of your [Onc
 
 ## Tested Devices
 
+- [20RESA]
```
Comment (bdraco) [inline source/_integrations/oncue.markdown]:
```suggestion
- 20RESA
```

**Review by bdraco — APPROVED** [review]
Thanks @rssalerno

**Comment by bdraco** [comment]
Oops. I should have renamed that to something more descriptive before merging

**Pull request merged**